resources/themes/pterodactyl/base/index.blade.php

```diff
@@ -62,7 +62,7 @@
                         <tr class="dynamic-update" data-server="{{ $server->uuidShort }}">
                             <td><code>{{ $server->uuidShort }}</code></td>
                             <td><a href="{{ route('server.index', $server->uuidShort) }}">{{ $server->name }}</a></td>
-                            <td>{{ $server->nodeName }}</td>
+                            <td>{{ $server->node_idName }}</td>
```

Comment:
as above, `node_id_name` or `node_name`, shouldn't it be `$server->node->name` actually?